docgen/layouts/common/meta.pug

```diff
@@ -24,6 +24,6 @@ html
     meta(content='Boost search on your React apps with Algolia', property='og:description')
     meta(content='React InstantSearch', property='og:site_name')
 
-    title React InstantSearch | #{title}
+    title React InstantSearch | Lightning Fast React and React Native Search
```

Comment:
Ah yep now probably not what we want, @neilrichler if you want that tagline only on the homepage, we can do that instead. Let us know.

Also the curren tagline on website homepage is "Lightning-fast search for React and React native apps" should we use that instead? In any case both should be consistent.